packages/note/src/css/index.js

```diff
@@ -0,0 +1,129 @@
+import core from '@pluralsight/ps-design-system-core'
+import { names as themeNames } from '@pluralsight/ps-design-system-theme/vars'
+
+export const resetButton = {
+  background: 'transparent',
+  border: 'none',
+  color: 'inherit',
+  font: 'inherit',
+  lineHeight: 'normal',
+  margin: 0,
+  overflow: 'visible',
+  padding: 0,
+  width: 'auto',
+
+  MozOsxFontSmoothing: 'inherit',
+  WebkitAppearance: 'none',
+  WebkitFontSmoothing: 'inherit',
+
+  '&::-moz-focus-inner': {
+    border: 0,
+    padding: 0
+  }
+}
+
+export default {
+  '.psds-note__list': {
+    listStyle: 'none',
+    marginLeft: 0
+  },
+
+  '.psds-note': {
+    alignItems: 'flex-start',
+    display: 'flex',
+    fontWeight: core.type.fontWeightMedium,
+    lineHeight: core.type.lineHeightStandard,
+    padding: `${core.layout.spacingLarge} ${core.layout.spacingMedium}`,
+
+    'li &': { borderTop: `1px solid ${core.colors.gray03}` },
+    'li:first-of-type &': { borderTop: 'none' }
+  },
+  [`.psds-note.psds-theme--${themeNames.light}`]: {
+    borderColor: core.colors.gray01
+  },
+
+  '.psds-note__action-bar': {
+    color: core.colors.gray02,
+    display: 'flex',
+
+    'div + &': { marginLeft: core.layout.spacingMedium }
```

Comment:
Part of why I wonder about this nesting type is that I have a hard timing diving its meaning.  

Does this mean "When an actionBar is sibling to a metadatum?"  It took me a running around in the react code to see html hierarchy and tagNames to figure that out, if it's even correct. :)

The alternative would be something like `.psds-note__metadata + .psds-note__action-bar`, right?

What are your thoughts on the readability?

Reply:
yes i do think i could make these selectors more readable - i'll see what i can come up with